en/02_Developer_Guides/00_Model/07_Permissions.md

```diff
@@ -47,12 +53,80 @@ class MyDataObject extends DataObject
 }
 ```
 
+It is good practice to let extensions extend permissions unless you _explicitly_ want a very restrictive permissions model. This is already done by default in the implementations of these methods in `DataObject`.
+
+You might also want to validate that the parent class doesn't deny access for a given action.
+
+```php
+use SilverStripe\Security\Permission;
+
+class MyDataObject extends SomeParentObject 
+{
+    public function canView($member = null) 
+    {
+        // If any extension returns false, the result will be false
+        // otherwise if any extension returns true, the result will be true
+        $extended = $this->extendedCan(__FUNCTION__, $member);
+        // You could also explicitly check for false values, and ignore any true values, if you don't
+        // want extensions saying members CAN perform this action before you've done your own checks.
+        if ($extended !== null) {
+            return $extended;
+        }
+
+        // If no extensions return true or false, check for a specific permission here.
+        return Permission::check('CMS_ACCESS_CMSMain', 'any', $member);
+    }
+
+    public function canEdit($member = null) 
+    {
+        // If the parent class says the member can't perform this action, don't let them do it.
+        // Be careful though - if the parent class doesn't implement canEdit(), you will end up
```

Comment:
`DataObject` implements `canEdit()`. The parent class may not be `DataObject`, and may not implement the method itself. It will _inherit_ the method, but may not implement it.

Reply:
Done